delete-conda-venv.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash zsh
+
+# Get the current folder name.
+current_folder_name="${PWD##*/}"
+
+# Deactivate any current conda environments and delete the one named as the current folder name.
+read "?Do you really want to delete '$current_folder_name' environment?  [[y]/n]" yn
+    case $yn in
+        [Yy]* ) ;;
+        [Nn]* ) return;;
+		* ) ;;
+    esac
+echo "Deleting '$current_folder_name' environment..."
+conda deactivate && conda env remove -n $current_folder_name;
+
```

Comment:
```suggestion

echo "Deleting '$current_folder_name' environment..."

conda deactivate && conda env remove -n $current_folder_name;
```

Reply:
Just a bit of formatting, but I this is a good idea with the echo! 🎉 We should do something similar when installing the pip requirements in `create-conda-venv` since that takes a little longer and it looks like the terminal just hangs there.